packages/addons/addon-depends/libexif/package.mk

```diff
@@ -2,10 +2,11 @@
 # Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
 
 PKG_NAME="libexif"
-PKG_VERSION="0.6.21"
-PKG_SHA256="16cdaeb62eb3e6dfab2435f7d7bccd2f37438d21c5218ec4e58efa9157d4d41a"
+PKG_VERSION="0.6.22"
+PKG_SHA256="5048f1c8fc509cc636c2f97f4b40c293338b6041a5652082d5ee2cf54b530c56"
 PKG_LICENSE="LGPL"
-PKG_SITE="http://sourceforge.net/projects/libexif"
+PKG_SITE="https://libexif.github.io"
 PKG_URL="http://downloads.sf.net/sourceforge/$PKG_NAME/$PKG_NAME-$PKG_VERSION.tar.bz2"
+PKG_URL="https://github.com/libexif/libexif/releases/download/libexif-$(echo ${PKG_VERSION} | tr '.' '_')-release/$PKG_NAME-$PKG_VERSION.tar.xz"
```

Comment:
Two PKG_URLs.

Bash is required, so bashisms are ok. `${PKG_VERSION//_/.} ` should be able to avoid the subshell.

Reply:
Fixed. Thanks for the pointer @antonlacon